app/services/workflows/main_workflow.rb

```diff
@@ -10,7 +10,23 @@ def call(assessment:, applicant:, partner:)
                              else
                                NonPassportedWorkflow.call(assessment:, applicant:, partner:)
                              end
-        RemarkGenerators::Orchestrator.call(assessment, calculation_output.capital_subtotals.combined_assessed_capital)
+        # we can take the lower threshold from the first eligibility records as they are all the same
+        lower_capital_threshold = assessment.applicant_capital_summary.eligibilities.first.lower_threshold
```

Comment:
@starswan Is lower_capital_threshold came for applicant and partner?

Reply:
partner doesn't have 'eligibilities' - only applicant. (yes this is one confusing thing still present in the codebase)